gnss_lib_py/utils/ephemeris_downloader.py

```diff
@@ -23,35 +26,89 @@
+is downloaded (COD0MGXFIN). The CODE final SP3 solutions became
+available starting GPS week 1962 or Aug 13, 2017. The CODE final
+solution includes: GPS+GLO+GAL+BDS+QZS. More about the CODE solution can
+be found in their papers [6]_ [7]_.
+
+If the SP3 date requested is between GPS week 1690 and 1961 or between
+May 25, 2012 and Aug 12, 2017, then the CODE 'com' short name solution
+is downloaded. More about the CODE 'com' solution can be found in their
+papers [9]_ [10]_.
+
+CLK Older than Two Weeks
+------------------------
+If the CLK date requested is more than two weeks previous to the
+current date and GPS week 2113 or later, then the CODE final solution
+is downloaded (COD0MGXFIN). The CODE final CLK solutions became
+available starting GPS week 2113 or July 5th, 2020. The CODE final
+solution includes: GPS+GLO+GAL+BDS+QZS. More about the CODE solution can
+be found in their papers [6]_ [7]_.
+
+If the CLK date requested is between GPS week 2034 and 2112 or between
+Jan 1, 2019 and July 4th, 2020, then the Wuhan University final
+solution is downloaded (WUM0MGXFIN). More about the WUM0MGXFIN solution
+can be found in their documention [11]_.
+
+If the CLK date requested is week GPS week 2034 and between Dec 30, 2018
+and Dec 31, 2018, then the GFZ short name solution is downloaded (gbm).
+More about the GFZ 'gbm' solution can be found in their
+documentation [13]_ [14]_.
+
+If the CLK date requested is between GPS week 1962 and 2033 or between
+Aug 13, 2017 and Dec 29th, 2018 then the Wuhan University short name
+solution is downloaded (wum). More about the "wum" solution can be found
+in their paper [12]_.
+
+If the CLK date requested is between GPS week 1710 and 1961 or between
+Oct 14, 2012 and Aug 12, 2017, then the CODE 'com' short name solution
+is downloaded. More about the CODE 'com' solution can be found in their
+papers [9]_ [10]_.
+
+IGS Resources
+-------------
+
+IGS network station information can be found at [3]_. IGS files can be
+viewed online using their file browser [5]_.
 
 References
 ----------
 .. [1] https://cddis.nasa.gov/Data_and_Derived_Products/GNSS/daily_30second_data.html
 .. [2] https://igs.org/mgex/data-products/#bce
 .. [3] https://network.igs.org/
 .. [4] https://igs.org/mgex/data-products/#orbit_clock
+.. [5] https://igs.bkg.bund.de/browseFiles
+.. [6] http://dx.doi.org/10.1016/j.asr.2020.04.038
+.. [7] http://dx.doi.org/10.7892/boris.75882.3
+.. [8] https://www.gfz-potsdam.de/en/section/space-geodetic-techniques/projects/mgex
+.. [9] http://doi.org/10.1007/1345_2015_161
+.. [10] http://doi.org/10.1007/s00190-016-0968-8
+.. [11] https://files.igs.org/pub/center/analysis/WUM0MGXFIN.acn
+.. [12] http://dx.doi.org/10.1007/s00190-015-0862-9
+.. [13] https://files.igs.org/pub/resource/pubs/workshop/2014/Workshop%202014%20-%20PS11%20-%20Deng%20-%202226%20-%20Precise%20orbit%20determination%20of%20Beidou%20Satellites%20at%20GFZ.pdf
+.. [14] http://doi.org/10.1007/1345_2015_120
 
 """
 
```

Comment:
In the two lines below, do we need to update the date?

Reply:
Our standard is that the date at the top of each file is when the file itself was first created.